### PR TITLE
:memo: Fix typo of exp_lpd_point variable name in stacking function

### LINKED ISSUE #1410: Typo in the example Bayesian Hierarchical Stacking
Hi, I noted that is a typo in the Bayesian Hierarchical Stacking notebook. In function ```stacking``` (cell 16), line 34, the code said:

```K = lpd_point.shape[1]  # number of candidate models```

However, the function argument is ```exp_lpd_point```. The example runs because Python allows functions to use variables outside its scope. The same type appears in the same function on line 62:

``` logp = jax.nn.logsumexp(lpd_point + log_w, axis=1)```

Best

### PR DESCRIPTION
**Pull request by cpieringer**
I fixed the typo example ```bayesian_hierarchical_stacking.ipynb``` function ```stacking```, lines 42 and 70.

Fixes #1410

### CODE REVIEW

File: notebooks/source/bayesian_hierarchical_stacking.ipynb [hunk 1]
```diff
@@ -800,7 +800,7 @@
     "    N = X.shape[0]\n",
     "    d = X.shape[1]\n",
     "    N_test = X_test.shape[0]\n",
-    "    K = lpd_point.shape[1]  # number of candidate models\n",
+    "    K = exp_lpd_point.shape[1]  # number of candidate models\n",
```
Comment (fehiepsi) [inline notebooks/source/bayesian_hierarchical_stacking.ipynb:803]: I think we need to use `lpd_point` as the input of the model, rather than `exp_lpd_point`. Later, we will take exponential of `lpd_point` at `jax.nn.logsumexp(lpd_point + log_w, axis=1)`

Comment (cpieringer) [reply notebooks/source/bayesian_hierarchical_stacking.ipynb:803]: Ok, I see. I think it is enough to change the name at the function definition.

Comment (fehiepsi) [reply notebooks/source/bayesian_hierarchical_stacking.ipynb:803]: You are right. It seems that we don't use `exp_lpd_point` in the colab. Do you want to fix the typo in this PR?

Comment (cpieringer) [reply notebooks/source/bayesian_hierarchical_stacking.ipynb:803]: I am trying to fix it, but, it is already fixed. I do not know if it was my pull request or the request of another contributor.  I send a message through the issue.
